app/views/layouts/application.html.erb

```diff
@@ -8,12 +8,26 @@
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
+
+    <%= stylesheet_link_tag "sticky-footer-navbar" %>
```

Comment:
Here you are importing other CSS files:

https://github.com/kc-si/JSCLook/blob/d5f85db828a57858bd02f5e91129af38a48d2c68/app/assets/stylesheets/application.bootstrap.scss#L1-L2

Maybe it is possible to import `sticky-footer-navbar.css` the same way?

Reply:
You mean to use importmap and import from internet? or from my localization but just this method?